### PR TITLE
Disable coverage temporarily

### PR DESCRIPTION
**Pull request by vanbasten23**
Current post-submit CPU CI is failing with an error:
```
INFO: Streaming build results to: https://source.cloud.google.com/results/invocations/19daf24a-3522-43d4-8a7f-87f9a96dc373
+ cp /tmp/pytorch/xla/bazel-out/_coverage/_coverage_report.dat /tmp/merged.dat
+ genhtml /tmp/merged.dat -o /home/jenkins/htmlcov/cpp/cpp_lcov.info
Reading data file /tmp/merged.dat
Resolved relative source file path "external/triton/third_party/nvidia/include/TritonNVIDIAGPUToLLVM/PTXAsmFormat.h" with CWD to "/tmp/pytorch/xla/external/triton/third_party/nvidia/include/TritonNVIDIAGPUToLLVM/PTXAsmFormat.h".
Found 258 entries.
Found common filename prefix "/tmp/pytorch/xla/torch_xla"
Writing .css and .png files.
Generating output.
genhtml: ERROR: cannot read /tmp/pytorch/xla/external/triton/third_party/nvidia/include/TritonNVIDIAGPUToLLVM/PTXAsmFormat.h
Processing file /tmp/pytorch/xla/external/triton/third_party/nvidia/include/TritonNVIDIAGPUToLLVM/PTXAsmFormat.h
Error: Process completed with exit code 2.
```

### CODE REVIEW

**Comment by vanbasten23** [comment]
Ok, the above commit [c98edf5](https://github.com/pytorch/xla/pull/6746/commits/c98edf55461e16179bc7a4a948af518412971a40) can reproduce the failure.

**Comment by vanbasten23** [comment]
Let me disable the genhtml first.

File: .circleci/common.sh [hunk 1]
```diff
@@ -198,8 +198,10 @@ function run_torch_xla_cpp_tests() {
         PJRT_DEVICE=CPU test/cpp/run_tests.sh $EXTRA_ARGS -L""
         cp $XLA_DIR/bazel-out/_coverage/_coverage_report.dat /tmp/merged.dat
       fi
-      genhtml /tmp/merged.dat -o ~/htmlcov/cpp/cpp_lcov.info
-      mv /tmp/merged.dat ~/htmlcov/cpp_lcov.info
+      # The command below is failing the post-submit CPU CI
```
Comment (yeounoh) [inline .circleci/common.sh:201]: Ok, feel free to land after finish testing.

**Review by yeounoh — APPROVED** [review]
LGTM

**Pull request merged**